CODEOWNERS

```diff
@@ -0,0 +1,21 @@
+# Code Ownership for buzzmobile.
+# Order matters. The last match is used, so more specific owners should be 
+# last.
+
+# Default owners/reviewers for anything.
+* @irapha @joshuamorton
+
+# Language specific reviewers:
+*.py @iraphael @joshuamorton
+*.cpp @chsahit
+
+# Folder and package level ownership.
+ci_scripts/* @irapha @joshuamorton
+
+# Buzzmobile subpackage overrides.
+buzzmobile/simulation/* @chsahit
+buzzmobile/tests/* @joshuamorton
```

Comment:
Does this mean that anytime anyone makes a test you specifically will have to review? I like this idea right now (because in general any test will be using pyros and will require your attention to make sure things are right) but I can see this being a problem in the future when we have a lot of tests (ha)